pyproject.toml

```diff
@@ -119,15 +119,12 @@ ignore = [
   "D104",  # Missing docstring in public package
   "D105",  # Missing docstring in magic method
   "D107",  # Missing docstring in `__init__`
-  "E501",  # line too long | Black take care of it
-
+  "E501",  # line too long
```

Comment:
This one was here to avoid black and ruff to colide in their appreciation of line length, if we exclusively rely on ruff you can drop this and accept all suggestion/fix